specification/mediaservices/resource-manager/Microsoft.Media/stable/2018-07-01/AssetsAndAssetFilters.json

```diff
@@ -0,0 +1,1264 @@
+        "storageEncryptionFormat": {
+          "type": "string",
+          "enum": [
+            "None",
+            "MediaStorageClientEncryption"
+          ],
+          "x-ms-enum": {
+            "name": "AssetStorageEncryptionFormat",
+            "values": [
+              {
+                "value": "None",
+                "description": "The Asset does not use client-side storage encryption (this is the only allowed value for new Assets)."
+              },
+              {
+                "value": "MediaStorageClientEncryption",
+                "description": "The Asset is encrypted with Media Services client-side encryption."
+              }
+            ],
+            "modelAsExtensible": true
+          },
+          "description": "The Asset encryption format. One of None or MediaStorageEncryption.",
+          "readOnly": true,
+          "x-nullable": false
+        }
+      },
+      "type": "object",
+      "description": "The Asset properties."
+    },
+    "AssetContainerSas": {
+      "properties": {
+        "assetContainerSasUrls": {
+          "type": "array",
+          "items": {
+            "type": "string"
+          },
+          "description": "The list of Asset container SAS URLs."
+        }
+      },
+      "type": "object",
+      "description": "The Asset Storage container SAS URLs."
+    },
+    "AssetFileEncryptionMetadata": {
+      "properties": {
+        "initializationVector": {
+          "type": "string",
+          "description": "The Asset File initialization vector."
+        },
+        "assetFileName": {
+          "type": "string",
+          "description": "The Asset File name."
+        },
+        "assetFileId": {
+          "type": "string",
+          "format": "uuid",
+          "description": "The Asset File Id."
+        }
+      },
+      "type": "object",
+      "required": [
+        "assetFileId"
+      ],
+      "description": "The Asset File Storage encryption metadata."
+    },
+    "StorageEncryptedAssetDecryptionData": {
+      "properties": {
+        "key": {
+          "type": "string",
+          "format": "byte",
+          "description": "The Asset File storage encryption key."
+        },
+        "assetFileEncryptionMetadata": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/AssetFileEncryptionMetadata"
+          },
+          "description": "Asset File encryption metadata."
+        }
+      },
+      "type": "object",
+      "description": "Data needed to decrypt asset files encrypted with legacy storage encryption."
+    },
+    "ListStreamingLocatorsResponse": {
+      "properties": {
+        "streamingLocators": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/AssetStreamingLocator"
+          },
+          "description": "The list of Streaming Locators."
+        }
+      },
+      "type": "object",
+      "description": "The Streaming Locators associated with this asset."
```

Comment:
are properties of `ListStreamingLocatorsResponse` and `AssetStreamingLocator` eligible to be marked as readonly? they are used to represents result of `Assets.listStreamingLocators` api and not used as payload for any create|update api.

Reply:
Yes, I marked those as readonly.